doc/sflow/sflow_hld.md

```diff
@@ -183,13 +183,14 @@ When sflow is disabled globally, sampling is stopped on all relevant interfaces
 
   Configure the sample-rate for a specific interface. 
 
-  The default sample rate for any interface is (ifSpeed / 1e6) where ifSpeed is in bits/sec.  So, the default sample rate based on interface speed is:
+  The default sample rate for any interface is (ifSpeed / 1e7) where ifSpeed is in bits/sec.  So, the default sample rate based on interface speed is:
 
-  * 1-in-1000 for a 1G link
-  * 1-in-10,000  for a 10G link
-  * 1-in-40,000  for a 40G link
-  * 1-in-50,000  for a 50G link
-  * 1-in-100,000  for a 100G link
+  * 1-in-100 for a 1G link
+  * 1-in-1,000  for a 10G link
+  * 1-in-4,000  for a 40G link
+  * 1-in-5,000  for a 50G link
+  * 1-in-10,000  for a 100G link
+  * 1-in-40,000  for a 400G link

```

Comment:
Other changes look good. w.r.t the sample rate, discussed with @prsunny and @dgsudharsan. The HLD is good (i.e. the default sampling rate (of 1 in N packets sampled) should be (ifSpeed (in bits per sec) / 1e6).  This is based on recommendation from InMon (as described in https://blog.sflow.com/2013/06/large-flow-detection.html  : check the Link Speed/Sampling Rate table).  Can you please raise PR for https://github.com/Azure/sonic-swss/blob/321291aa7aca89a3ccef2bed36aff074eac1054f/cfgmgr/sflowmgr.h#L21 to change the SFLOW_SAMPLE_RATE_VALUE_* to be (ifSpeed / 1e6) ?

Reply:
Sure, I will raise another PR for this.

Reply:
Hi @padmanarayana , I have raised a PR: [sonic-swss#1623](https://github.com/Azure/sonic-swss/pull/1623).
I also restored the default sample rate to "ifSpeed / 1e6" in this PR.